MAINTENANCE.md

```diff
@@ -0,0 +1,80 @@
+# Roles, People, and Processes for Maintaining the Tagging Schema
+
+This document outlines how this project is maintained.
+
+## Roles & People
+
+### Maintainer Role
+
+[Martin](https://github.com/tyrasd) maintains this project as part of his work on the iD Editor project.
+
+The maintainer role includes:
```

Comment:
Are they also approving people to "Contributor" role?

Reply:
Good point, added